lib/bolt/outputter/human.rb

```diff
@@ -123,13 +123,15 @@ def print_task_info(task)
       end
 
       def print_plan(result)
-        # If a hash or array, pretty-print as JSON
-        if result.is_a?(Hash) || result.is_a?(Array)
-          if result.empty?
-            # Avoids extra lines for an empty result
+        # If the object has a json representation display it
+        if result.respond_to?(:to_json)
```

Comment:
This will be true for almost everything, since the json library monkey-patches things.

We could special-case String. It's probably not important to address right now though.

Reply:
Yeah I was thinking long term we may want to have some convention like "_output" for the string summary of a plan.